test/e2e/scalability/load.go

```diff
@@ -566,8 +566,16 @@ func scaleResource(wg *sync.WaitGroup, config testutils.RunObjectConfig, scaling
 		LabelSelector:   selector.String(),
 		ResourceVersion: "0",
 	}
-	_, err := config.GetClient().CoreV1().Pods(config.GetNamespace()).List(options)
-	framework.ExpectNoError(err, fmt.Sprintf("listing pods from rc %v", config.GetName()))
+	attemptsLeft := 5
+	for ; attemptsLeft > 0; attemptsLeft-- {
+		_, err := config.GetClient().CoreV1().Pods(config.GetNamespace()).List(options)
+		if err == nil {
+			break
+		}
+		framework.Logf("Failed to list pods from RC %v due to: %v", config.GetName(), err)
```

Comment:
Not necessarily RC, use `config.GetKind()` here.

Reply:
Done. Also changed it above.